Readme.md

```diff
@@ -18,3 +18,113 @@ like to [backbone](https://github.com/jashkenas/backbone), but with data streams
 
 Then main goal of framework is to create universal components, which must render on server and client
 and provide easy way to restore their state to continue work.
+
+## Template render pragma
+
+Malanka supports special pragma for rendering components. It is use only during compilation and 
+control rendering process
+
+### #bundle
+
+It is extract component to separate module with webpack `promise-loader`:
+
+```handlebars
+<div class="test">
+    <Sidebar #bundle=true>
+        <Navigation items=navigationItems>
+            ...
+        </Navigation>
+    </Sidebar>
+</div>
+```
+
+In this example `Sidebar` and `Navigation` will be extracted in separate module and will be
+rendered on server as normal components, but initialize in Browser only after module loading.
+If rendering on client side only, then it appear on screen only after module loading.
```

Comment:
Also context of it is not so clear, can you clarify is this a component or module?